backends-velox/src/main/scala/org/apache/gluten/config/VeloxConfig.scala

```diff
@@ -532,6 +535,16 @@ object VeloxConfig {
       .booleanConf
       .createWithDefault(false)
 
+  val VELOX_BROADCAST_BUILD_HASHTABLE_ONCE_PER_EXECUTOR =
+    buildConf("spark.gluten.velox.buildHashTableOncePerExecutor.enabled")
+      .internal()
+      .doc(
+        "Experimental: When enabled, the hash table is " +
+          "constructed once per executor. If not enabled, " +
+          "the hash table is rebuilt for each task.")
+      .booleanConf
+      .createWithDefault(true)
```

Comment:
Should be turned off by default given it's experimental?

Reply:
@zhztheplayer We plan to enable this feature by default, as we have successfully passed all unit tests except for this one https://github.com/apache/incubator-gluten/issues/9342.